mycore-base/src/main/java/org/mycore/backend/jpa/MCREntityTransaction.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.backend.jpa;
+
+import org.mycore.common.MCRSessionTransaction;
+
+public class MCREntityTransaction implements MCRSessionTransaction {
```

Comment:
I am "out" - the naming is confusing for me:
MCREntityTransaction / MCRSessionTransaction 

Do we really implement our own/new Transaction or are we just managing an existing JPA-Transaction here?

Reply:
MCRSessionTransaction could also be named MCRDBTransaction or MCRDatabaseTransaction, while MCREntityTransaction fits well I think.